test/conditionals_test.coffee

```diff
@@ -155,6 +155,23 @@ describe 'ConditionChecker', ->
     beforeEach ->
       @fr = new FormRenderer Fixtures.FormRendererOptions.SHORT()
 
+    it 'initializes directly from a field', ->
+      conditional =
+        method: 'eq'
+        value: '123'
+        response_field_id: '35'
+
+      field = @fr.response_fields.get(conditional.response_field_id)
+
+      checker = new FormRenderer.ConditionChecker(null, conditional, field)
+      expect(checker.isVisible()).to.eql(false)
+
+      field.setExistingValue('123')
+
+      checker = new FormRenderer.ConditionChecker(null, conditional, field)
+      expect(checker.isVisible()).to.eql(true)
```

Comment:
And, like the above, should we add the same 
```
expect(checker.isDefaultHidden()).to.eql(true)
```
to make sure `isDefaultHidden` doesn't change once a field has a value (or meets conditions that would show it)?

Reply:
agree for both of your comments in this spec.

Reply:
actually, that won't work because `isDefaultHidden` is defined on the ResponseFieldModel, not the ConditionChecker class

Reply:
I was pretty surprised that there wasnt a concrete set of tests for the ResponseField model